src/Braze.ts

```diff
@@ -79,7 +79,7 @@ export class Braze {
       list: () => catalogs.synchronous.listCatalogs(this.apiUrl, this.apiKey),
       items: <T extends catalogs.synchronous.CatalogListItem>(
         body: catalogs.synchronous.CatalogListItemsBody,
-      ) => catalogs.synchronous.listCatalogItems<T>(this.apiUrl, this.apiKey, body),
+      ) => catalogs.synchronous.getListCatalogItemsIterator<T>(this.apiUrl, this.apiKey, body),
```

Comment:
This would make this endpoint not backwards compatible, so not sure if that's a problem. Should I add something to change the version number differently?

Reply:
If it's a `BREAKING CHANGE`, then we need to add that to the [commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples) and it will bump the major version